linux_os/guide/system/permissions/files/permissions_important_account_files/file_groupowner_backup_etc_group/rule.yml

```diff
@@ -16,8 +16,7 @@ identifiers:
     cce@rhel8: CCE-83475-4
 
 references:
-    cis@rhel7: 6.1.8
-    cis@rhel8: 6.1.8
+    cis: 6.1.8
```

Comment:
Since other section `6.1.x` rules differ between RHEL and Ubuntu, I might suggest we keep these two separate for the time being.